addons/sale/data/product_demo.xml

```diff
@@ -184,25 +184,15 @@
         <field name="weight">0.01</field>
     </record>
 
-    <record id="product_template_attribute_exclusion_1" model="product.template.attribute.exclusion">
-        <field name="product_tmpl_id" ref="product.product_product_4_product_template" />
-        <field name="value_ids" eval="[(6,0,[ref('product.product_4_attribute_2_value_2')])]"/>
-    </record>
-
     <record id="product_template_attribute_exclusion_2" model="product.template.attribute.exclusion">
         <field name="product_tmpl_id" ref="product.product_product_11_product_template" />
         <field name="value_ids" eval="[(6,0,[ref('product.product_11_attribute_1_value_1')])]"/>
     </record>
 
     <!--
     The "Customizable Desk's Aluminium" attribute value will excude:
-    - The "Customizable Desk's Black" attribute
     - The "Office Chair's Steel" attribute
      -->
-    <record id="product.product_4_attribute_1_value_2" model="product.template.attribute.value">
-        <field name="exclude_for" eval="[(6,0,[ref('sale.product_template_attribute_exclusion_1') ,ref('sale.product_template_attribute_exclusion_2')])]" />
-    </record>
-
     <record id="product_template_attribute_exclusion_3" model="product.template.attribute.exclusion">
         <field name="product_tmpl_id" ref="product.product_product_11_product_template" />
         <field name="value_ids" eval="[(6,0,[ref('product.product_11_attribute_1_value_2')])]"/>
```

Comment:
IMHO, would be cleaner if you moved all the exclusions at once, along with the comments explaining them.
(aka `product_template_attribute_exclusion_2`, `product_template_attribute_exclusion_3` + the change to `product_4_attribute_1_value_1`)